gooey/internal/scrollbar.lua

```diff
@@ -54,14 +56,13 @@ function M.vertical(handle_id, bounds_id, action_id, action, fn, refresh_fn)
 
 	scrollbar.enabled = core.is_enabled(handle)
 	scrollbar.node = handle
-	scrollbar.node_id = node_id
+	scrollbar.node_id = handle_id
+	scrollbar.refresh_fn = refresh_fn
 	scrollbar.bounds = bounds
 	scrollbar.bounds_size = bounds_size
 	scrollbar.handle_size = handle_size
 
 	if action then
-		scrollbar.refresh_fn = refresh_fn
```

Comment:
*thought*: We were conditionally assigning this, so the `scrollbar.refresh()` call below wasn't doing anything on init.

Reply:
Nice catch!